packages/iov-ethereum/src/abi.ts

```diff
@@ -12,6 +12,11 @@ export interface HeadTail {
   /** Arguments split by positions as defined by head */
   readonly tail: ReadonlyArray<Uint8Array>;
 }
+export enum SwapContractEvent {
+  Opened = "Opened",
+  Claimed = "Claimed",
+  Aborted = "Aborted",
```

Comment:
Are the raw values of this enum ever used? If now, we should drop them. If yes, we should document why the need to have a specific value.